app/konnect/reference/search.md

```diff
@@ -0,0 +1,128 @@
+| Select from a list of available entities | For example, if you are attaching a route to a service, you can use search to find and identify which service to attach the route to. Or when you're adding members to a team, you can search for the user. |
+
+## Query Syntax
+
+The {{site.konnect_short_name}} Search API provides selectors, reserved characters, and logical operators that you can use to narrow your entity search. By combining these selectors, reserved characters, and logical operators, you can construct complex and precise queries to effectively use the {{site.konnect_short_name}} Search API.
+
+To perform a simple search, you can just search by the name of an entity, like a service, API product, or name of a team. You can also perform an advanced search using {{site.konnect_short_name}}'s query syntax to get more granular results.
+
+The following is an example advanced search query syntax:
+
+```
+type:team AND NOT label.department:eng AND name:*_qa
+```
+The following provides more details about the different components of the query syntax:
+* Selectors: `type`, `label`, and `name`. They define what you are searching by. 
+* Entity type: `team`. These define what {{site.konnect_short_name}} entity you want to search for.
+* Logical operator: `AND NOT` and `AND`. These are used to combine multiple criteria in a query.
+* Wildcard: `*`
+* Search values: `eng` and `_qa`. These values aren't set and can vary. They determine what you search for, such as the name of an API product or service that is unique to your environment.
+
+### Entity types
+
+The following {{site.konnect_short_name}} entity types are supported: 
+
+- `api_product`  
+- `api_product_version`  
+- `application`  
+- `ca_certificate`  
+- `certificate`  
+- `consumer`  
+- `consumer_group`  
+- `control_plane`  
+- `control_plane`  
+- `data_plane`  
+- `developer`  
+- `developer_team`  
+- `key`  
+- `key_set`   
+- `mesh`  
+- `mesh_control_plane`  
+- `plugin`  
+- `portal`  
+- `report`  
+- `route`  
+- `service`  
+- `sni`  
+- `system_account`  
+- `target`  
+- `team`  
+- `upstream`  
+- `user`  
+- `vault`  
+- `zone`  
+
+Additional entities may be added in future releases. You can view a list of all the supported entities by sending the following API request:
+
+```bash
+curl -X 'GET' \
+  'https://global.api.konghq.com/v1/search/types' \
+  -H 'accept: application/json'
+```
+
+#### Searchable attributes
+For each entity type, there is a list of entity specific attributes that are searchable. These attributes are returned in the attributes object in the search response while the schema of the searchable attributes can be found in the types endpoint. <!-- I'm guessing this specifies stuff like "department" in the example, but I wasn't finding this listed by the types endpoint. Also, can you search by a certain entity? Like "user"?-->
```

Comment:
@haydenlam I'm guessing this (searchable attributes) specifies stuff like "department" in the example, but I wasn't finding this listed by the types endpoint. Also, can you search by a certain entity? Like "user"?

Reply:
Yes, you can search by entity type such as "user". The attribute "department" does not exist, but is a use case for tagging and labeling. So if users label their entities with department metadata, it can be searchable that way. I'll update the example with something more concrete.